src/js/controller.js

```diff
@@ -518,6 +518,20 @@ export const MediaUIStates = {
     mediaEvents: ['loadstart'],
     textTracksEvents: ['addtrack', 'removetrack', 'change'],
   },
+  MEDIA_CHAPTERS_CUES: {
+    get: function (controller) {
+      const { media } = controller;
+      if (!media) return [];
+
+      const [chaptersTrack] = getTextTracksList(media, { kind: TextTrackKinds.CHAPTERS });
+      if (chaptersTrack) chaptersTrack.mode = 'hidden';
```

Comment:
I don't understand this. could you explain more here? 

would we want to warn on each side effect in the code?

Reply:
No, but we're doing something that isn't required for the functionality of this state and may cause unexpected side effects.

For example, the `<track>` element and the corresponding `TextTrack` class instance will not display its contents, even when `mode === "showing"` (See: https://html.spec.whatwg.org/multipage/media.html#the-track-element & https://html.spec.whatwg.org/multipage/media.html#text-track-model). Additionally, folks may want to intentionally/conditionally set `mode = "disabled"` for chapters, in which case it would likely be better to simply filter those out.

If we're going to instead make this additional assumption in our architecture of forcing the first chapters track to specifically `mode = hidden`, it's under that case that I think we should have something like:
```js
// or alternatively only if  `chaptersTrack.mode === TextTrackModes.DISABLED`
if (chaptersTrack && chaptersTrack.mode !== TextTrackModes.HIDDEN) {
  console.warn('Media Chrome: changing Chapters TextTrack mode to "hidden" to ensure availability of Cues.");
  chaptersTrack.mode = 'hidden';
}
```
Happy to get thoughts/input from others, though, if you think this is wrongheaded.

Reply:
ah that makes sense, thanks for the explanation!
will make this more restrictive.

for chapters I'm more leaning to enable them if they are added since they're not so intrusive.


@heff do you have any thoughts on this one?

Reply:
I'm not quite understanding the common workflow here. In what cases is it needed for us to turn on chapters for people? Can we just rely on the existing APIs for turning on chapters, like the default attribute, and educating people that they need to include that? 

I can see the motivation for making this simple on people who might forget to turn them on by default. But then it also might complicate things down the road, like when there's multiple languages of chapters.

Reply:
using `default` attribute only doesn't seem to work but I could query the track element and compare... if it's the same one `trackElement.track == chaptersTrack`, set it to showing or hidden if the default attribute is present...

Reply:
this turned out to be a bug with custom-media-element, 
got that fixed here https://github.com/muxinc/custom-media-element/pull/18

now it works nicely with the track `default` attribute